P/Pythia8/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pythia8"
+version = v"8.244.0"
+
+# Collection of sources required to complete build
+sources = [
+    "http://home.thep.lu.se/~torbjorn/pythia8/pythia8244.tgz" =>
+    "e34880f999daf19cdd893a187123927ba77d1bf851e30f6ea9ec89591f4c92ca",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pythia8244/
+./configure --prefix=${prefix} --enable-shared --enable-64bit
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    Linux(:powerpc64le, libc=:glibc),
+    MacOS(:x86_64)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpythia8", :libpythia8)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"9.1.0")
```

Comment:
Is GCC 9 really required by this library?  Keep in mind https://github.com/JuliaPackaging/Yggdrasil/wiki/Build-Tricks#building-with-an-old-gcc-version-a-library-that-has-dependencies-built-with-newer-gcc-versions when you choose to use recent GCC versions.

Reply:
[This](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=1081&view=logs&j=0f152180-077b-50d8-339d-4d860b71551b&t=99fec6b7-45c0-5795-d783-6a06c518257a&l=1811) is the problem with compiling with a too recent version of GCC: the library can't just be dlopen'ed by many systems:
```
ERROR: could not load library "/workspace/P/Pythia8/build/x86_64-linux-gnu/qEoTbMrR/destdir/lib/libpythia8.so"
/usr/local/julia/bin/../lib/julia/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /workspace/P/Pythia8/build/x86_64-linux-gnu/qEoTbMrR/destdir/lib/libpythia8.so)
[...]
[ Info: Found a valid dl path libpythia8.so while looking for libpythia8
[ Info: /workspace/P/Pythia8/build/x86_64-linux-gnu/qEoTbMrR/destdir/lib/libpythia8.so matches our search criteria of libpythia8
[ Info: /workspace/P/Pythia8/build/x86_64-linux-gnu/qEoTbMrR/destdir/lib/libpythia8.so cannot be dlopen'ed
```

You have to choose the oldest version as possible, compatibly with the requirements of the library of course.

Reply:
Could probably be removed... Thanks for checking.